sr_demos/scripts/sr_demos/demo.py

```diff
@@ -154,345 +172,376 @@ def run(self):
+        self.commander.move_to_named_target("open")
+
+        for _ in range(0, 3):
+            for movement in flex_ext_sequence:
+                self.execute_command_check(movement, 0.2)
+
+        self.execute_command_check('pre_ff_ok', 1.0)
+        self.execute_command_check('ff_ok', 3.3)
+        self.execute_command_check('ne_wr', 1.1)
+        self.execute_command_check('nw_wr', 1.1)
+        self.execute_command_check('sw_wr', 1.1)
+        self.execute_command_check('se_wr', 1.1)
+        self.execute_command_check('ne_wr', 0.7)
+        self.execute_command_check('nw_wr', 0.7)
+        self.execute_command_check('sw_wr', 0.7)
+        self.execute_command_check('se_wr', 0.7)
+        self.execute_command_check('zero_wr', 0.4)
+        self.commander.move_to_named_target("open")
+        rospy.loginfo("Standard demo completed")
+
+    def rock_paper_scissors(self):
+        '''
+            Runs the Rock, Paper, Scissors demo
+        '''
+        rospy.loginfo("Rock, Paper, Scissors demo started")
+        self.commander.move_to_named_target("open")
+
+        rospy.loginfo("Welcome to the Rock, Paper, Scissors game!")
+        rospy.loginfo("The hand will count down from 3 and then you will make a gesture.")
+        rospy.loginfo("Rock is a fist, Paper is a flat hand, and Scissors is a peace sign.")
+        rospy.loginfo("Ready?")
+        rospy.sleep(3)
+
+        # Count down
+        self.execute_command_check('count_down_3', 1.0)
+        rospy.loginfo("3")
+        self.execute_command_check('count_down_2', 1.0)
+        rospy.loginfo("2")
+        self.execute_command_check('count_down_1', 1.0)
+        rospy.loginfo("1")
+        self.execute_command_check('count_down_0_ff', 1.0)
+        self.execute_command_check('count_down_0_th', 1.0)
+        rospy.loginfo("Make your gesture!")
+
+        # Select a pose at random
+        poses = ['rock', 'paper', 'scissors']
+        pose = random.choice(poses)
+        self.execute_command_check(pose, 1.0)
+        rospy.loginfo("The hand made the {} gesture!".format(pose))
+        rospy.sleep(5.0)
+
+        if pose == "rock":
+            self.commander.move_to_named_target("fingers_pack_thumb_open")
+        self.commander.move_to_named_target("open")
+        rospy.loginfo("Rock, Paper, Scissors demo completed")
+
+    def grasp_demo(self):
+        '''
+            Runs a demo that imitates grasping and squeezing an object.
+        '''
+        rospy.loginfo("Grasp Demo Started")
+
+        self.commander.move_to_named_target("open")
+        rospy.loginfo("The hand will now imitate grasping and squeezing an object...")
+        self.execute_command_check('pregrasp_pos', 2.0)
+        self.execute_command_check('grasp_pos', 5.0)
+
+        # Send all joints to current position to compensate
+        # for minor offsets created in the previous loop
+        hand_pos = {joint: degrees(i) for joint, i in robot.commander.get_joints_position().items()}
```

Comment:
```suggestion
        hand_pos = {joint: degrees(i) for joint, i in self.commander.get_joints_position().items()}
```

Reply:
fixed in 92637be0ff846ca31b76aae33b809ec2e931900b